f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml

```diff
@@ -942,6 +948,13 @@ Dow.prototype = {
         "DOW" 
       );
 
+      var requirementsCostList = Dow.prototype.findPackageRecords(
+        this.REQUIREMENTS_COST_EST_TABLE,
+        this.acquisitionPackage.sys_id
+      )
+      // Q: is there only one ref per pacakge for requirements costs?
```

Comment:
Yes, it is expected that each package should only have one record in Requirements Cost Estimate.

Reply:
Thanks for clearing that up.